src/js/stores/AuthStore.js

```diff
@@ -96,6 +96,20 @@ class AuthStore extends GetSetBaseStore {
     }
   }
 
+  getUserLabel() {
+    const user = this.getUser();
+
+    let userLabel = null;
+
+    if (user && !user.is_remote) {
+      userLabel = user.description;
+    } else if (user && user.is_remote) {
+      userLabel = user.uid;
+    }
+
+    return userLabel;
```

Comment:
🎨  FYI: we had a lot of discussions about code style lately discussing about the need of `let`, `if .. else` and ternaries.

That said, I'd love to have an early exit for `user == undefined` and then just `return user.is_remote ? user.uid : user.description;`

Reply:
Good call